sass/heidelberg/_heidelberg-plugin.sass

```diff
@@ -120,3 +102,18 @@ $_right: odd
 // Needs to be on own line to work in IE8
 .no-csstransforms3d .Heidelberg-Page.with-Spread.is-active + .Heidelberg-Page.with-Spread.is-active .Heidelberg-Spread
   left: -100%
+
+// Internet 10 only selector - checks Heidelberg HTMLElement attribute: data-useragent
+.Heidelberg-Book[data-useragent*='MSIE 10.0']
```

Comment:
Because this fixes a rendering issue only in IE 10.0 - All other version of IE are fine.